src/pages/standup/index.tsx

```diff
@@ -19,7 +19,7 @@ const StandUp: FC = memo(function StandUp() {
     const { dev } = router.query;
 
     const handleConditionalRendering = () => {
-        if (!isAuthenticating && isLoggedIn) {
+        if (!isAuthenticating) {
```

Comment:
isn't authenticating a type of loading? isLoggedIn false sounds more logical here

Reply:
fixed